workloads/test_format/test_format.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+# Initial sample workload for use with the testy framework.
+
+describe() {
+    echo "A test/format workload for use with the testy framework."
+}
+
+populate() {
+    echo "Populate is not implemented in this wokrload."
+}
+
+run() {
+    ${script_dir}/testy-metrics.sh workload_status 1
+    ${wt_build_dir}/test/format/format.sh -c ${workload_dir}/test_format/${format_config} -h ${database_dir} -j 1
```

Comment:
I think we could use `nproc` instead of `1` for the number of jobs.

Reply:
[4782b14](https://github.com/wiredtiger/testy/pull/82/commits/4782b141c4649c9daa8d542786779a5037f28af5)

Reply:
Doesn't seem to be the right commit id

Reply:
oops hehe